src/logger/index.js

```diff
@@ -0,0 +1,84 @@
+import winston from 'winston';
+
+const { combine, timestamp, align, printf, colorize } = winston.format;
+
+const alignedWithColorsAndTime = combine(colorize(),
+  timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+  align(),
+  printf(({ level, message, timestamp, serviceId, type }) => {
+    let prefix = '';
+    if (serviceId && type) {
+      prefix = `${serviceId} — ${type}`;
+    }
+    return `${timestamp} ${level.padEnd(15)} ${prefix.padEnd(55)} ${message}`;
+  }));
+
+const logger = winston.createLogger({
+  format: alignedWithColorsAndTime,
+  transports: [
+    new winston.transports.Console({
+      handleExceptions: true
+    }),
+  ],
+  exitOnError: false
```

Comment:
Why?
If set to `false`, then why `catch` everything at top level?
Let's discuss this🗣